test/audiomixcontroller/DefaultAudioMixController.test.ts

```diff
@@ -95,6 +95,14 @@ describe('DefaultAudioMixController', () => {
     expect(defaultAudioMixController.bindAudioDevice(device)).to.equal(true);
   });
 
+  it('can successfully set audio element and unbind element', () => {
```

Comment:
Nice try, but it won't test anything for two reasons.

#### Reason 1: DOMMockBuilder

In DOMMockBuilder, `setSinkId` just sets but returns immediately.

https://github.com/aws/amazon-chime-sdk-js/blob/218330d4046fd18b30cd488a12b0babee00a31ae/test/dommock/DOMMockBuilder.ts#L856-L862
  
Try adding the artificial delay. As you discovered, it will set `sinkId` and then take some time to finish. (Optional: You can even disable setting `srcObject` while this function is running to simulate the exact behavior of Chrome.)

```
setSinkId(deviceId: string): Promise<undefined> {
  this.sinkId = deviceId;
  await new Promise(resolve => setTimeout(resolve, mockBehavior.asyncWaitMs));
  return undefined;
}
```

#### Reason 2: Async

This `it` test will immediately finish after running `defaultAudioMixController.unbindAudioElement();`. Note that `bindAudioElement` and `unbindAudioElement` have the side effect unfortunately.

Convert your `it` to the `async` function, then add some delay, and then use `expect` to verify.

```
defaultAudioMixController.bindAudioElement(element);
defaultAudioMixController.unbindAudioElement();
await new Promise(resolve => new TimeoutScheduler(1000).start(resolve));

expect(element.srcObject).to.equal(null);
```